# Generated by Django 2.0.13 on 2019-05-14 09:21

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('accounts', '0001_initial'),
    ]

    operations = [
        migrations.AddField(
            model_name='user',
            name='first_name',
            field=models.CharField(blank=True, max_length=30, verbose_name='first name'),
        ),
        migrations.AddField(
            model_name='user',
            name='last_name',
            field=models.CharField(blank=True, max_length=150, verbose_name='last name'),
        ),
        migrations.AlterField(
            model_name='user',
            name='password',
            field=models.CharField(error_messages={'unique': 'パスワードが規則に従っていません'}, help_text='8文字以上で入力してください', max_length=150, validators=[django.core.validators.MinLengthValidator(8)], verbose_name='password'),
        ),
    ]
